src/urllib3/util/__init__.py

```diff
@@ -11,6 +11,7 @@
     PROTOCOL_TLS,
     SSLContext,
     assert_fingerprint,
+    is_ipaddress,
```

Comment:
`urllib3.util` are public APIs, are we okay with having this function be public?

Reply:
Thanks, fixed! I hadn't realized that `util` was more public than `util.ssl_`. I certainly don't want to add more public APIs